### PR TITLE
feat: Added snackbar

### PR DESCRIPTION
**Pull request by krishanu2911**
Added snackbar with js included with the variations of leading snackbar and baseline snackbar

https://user-images.githubusercontent.com/75560307/152867209-13017d41-8e12-4bb3-b81d-cf5933d83d54.mp4

### CODE REVIEW

File: JS/snackbar.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const leadingClose = document.querySelector("#leading-dismiss");
+const leadingDiv = document.querySelector("#leading-div");
+const baselineBtn = document.querySelector("#baseline");
+const baselineClose = document.querySelector("#baseline-dismiss");
+const baselineDiv = document.querySelector("#baseline-div");
+
+leadingBtn.addEventListener("click", leadingClickHandler);
+
+function leadingClickHandler(){
+    console.log("hello");
```
Comment (tanishq20) [inline JS/snackbar.js]: remove `console.log`
(Apply to all)

**Review by tanishq20 — COMMENTED** [review]
suggested small change

**Review by rittik-ghosh — COMMENTED** [review]
good to merge.

File: JS/snackbar.js (hunk 1 above)
Comment (krishanu2911) [reply JS/snackbar.js]: Done

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**